windowsdoc.md

```diff
@@ -127,6 +128,15 @@ provide a custom timeout period using `--eval-timeout` argument. So if you face
 this error, the best option is to install Nodejs on your local system. If you
 can't then use the `--eval-timeout` argument and set a higher timeout value.
 
+### Fatal error occurs when executing a python command Line instruction as a job on python version 3.
+
+If your workflow creates a python command line instruction (like we do in this
+[unit test](https://github.com/common-workflow-language/cwltool/blob/master/tests/test_examples.py#L117) ) and you get a error `Fatal Python error: Failed to initialize Windows
```

Comment:
If we run test_examples.test_partial_scatter, a python command line instruction is generated as below. Here I got this error.
```
Resolved 'E:\GitHub\gsoc\cwltool\tests\..\tests\wf\scatterfail.cwl' to 'file:///E:/GitHub/gsoc/cwltool/tests/wf/scatterfail.cwl'
[workflow scatterfail.cwl] start
[step step1] start
[workflow step1] start
[step sstep1] start
[jobee sstep1] c:\users\kapilk~1\appdata\local\temp\tmpxnohco$ python \
    -c \
    'from __future__ import print_function
import sys
print(sys.argv[1])
if sys.argv[1] == "2":
  exit(1)
else:
  f = open("foo"+sys.argv[1]+".txt", "wb")
  content = sys.argv[1]+"\n"
  f.write(content.encode('"'"'utf-8'"'"'))
if sys.argv[1] == "5":
  exit(1)
' \
    1
1
after
[jolb sstep1] completed success
[step sstep1] completed success
[step sstep2] start
[jobee sstep2] c:\users\kapilk~1\appdata\local\temp\tmp0rjlug$ cat \
    /C/users/kapilk~1/appdata/local/temp/tmpnzk_wf/stgae130961-a089-4793-b067-019a7abdd6b3/foo1.txt
cat: /C/users/kapilk~1/appdata/local/temp/tmpnzk_wf/stgae130961-a089-4793-b067-019a7abdd6b3/foo1.txt: No such file or directory
after
[joboo sstep2] completed permanentFail
[step sstep2] completed permanentFail
[workflow step1] completed permanentFail
[step step1] start
[workflow step1_2] start
[step sstep1_2] start
[jobee sstep1_2] c:\users\kapilk~1\appdata\local\temp\tmp32cq41$ python \
    -c \
    'from __future__ import print_function
import sys
print(sys.argv[1])
if sys.argv[1] == "2":
  exit(1)
else:
  f = open("foo"+sys.argv[1]+".txt", "wb")
  content = sys.argv[1]+"\n"
  f.write(content.encode('"'"'utf-8'"'"'))
if sys.argv[1] == "5":
  exit(1)
' \
    2
2
[job sstep1_2] Job error:
E:GitHub\gsoc\cwltool\tests\wf\echo.cwl:20:3: Error collecting output for parameter 'out':
E:GitHub\gsoc\cwltool	ests\wf\echo.cwl:23:7:   Did not find output file with glob pattern: '['foo2.txt']'
[joboo sstep1_2] completed permanentFail
[step sstep1_2] Output is missing expected field file:///E:/GitHub/gsoc/cwltool/tests/wf/scatterfail.cwl#step1/subtool/sstep1/out
[step sstep1_2] completed permanentFail
[workflow step1_2] completed permanentFail
[step step1] start
[workflow step1_3] start
[step sstep1_3] start
[jobee sstep1_3] c:\users\kapilk~1\appdata\local\temp\tmpdwpn_3$ python \
    -c \
    'from __future__ import print_function
import sys
print(sys.argv[1])
if sys.argv[1] == "2":
  exit(1)
else:
  f = open("foo"+sys.argv[1]+".txt", "wb")
  content = sys.argv[1]+"\n"
  f.write(content.encode('"'"'utf-8'"'"'))
if sys.argv[1] == "5":
  exit(1)
' \
    3
3
after
[jolb sstep1_3] completed success
[step sstep1_3] completed success
[step sstep2_3] start
[jobee sstep2_2] c:\users\kapilk~1\appdata\local\temp\tmpzly48y$ cat \
    /C/users/kapilk~1/appdata/local/temp/tmpswvggv/stg906b6972-c95c-4bf0-9029-0f877f8bfe8e/foo3.txt
cat: /C/users/kapilk~1/appdata/local/temp/tmpswvggv/stg906b6972-c95c-4bf0-9029-0f877f8bfe8e/foo3.txt: No such file or directory
after
[joboo sstep2_2] completed permanentFail
[step sstep2_3] completed permanentFail
[workflow step1_3] completed permanentFail
[step step1] completed permanentFail
[workflow scatterfail.cwl] completed permanentFail


========================== 1 passed in 10.48 seconds ==========================

Process finished with exit code 0
```

Sorry about the line wrap. I will fix it.